src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -1371,6 +1653,7 @@ end
                 )
             end
         else
+            # TODO: What is this for?
```

Comment:
This is where it is used:
https://github.com/CliMA/ClimateMachine.jl/blob/b2161c2313b90e707a63a3a1d3d2556a2d6ce57e/src/Atmos/Model/bc_energy.jl#L118-L176